examples/svsm-register.rs

```diff
@@ -34,8 +34,8 @@ fn main() -> anyhow::Result<()> {
 
     let config = ProxyArgs::parse();
 
-    let cr = ClientRegistration::new(config.workload.clone());
-    let registration = cr.register(&hex::decode(config.measurement)?, config.passphrase);
+    let cr = ClientRegistration::new(&hex::decode(config.measurement)?, config.passphrase);
+    let registration = cr.register();
 
     info!(
         "Registering workload {0} at {1}",
```

Comment:
I'm using `svsm-register` to register the attestation.
IIUC we will receive and ID that we should put in `host_data`, should we print it?

Reply:
Sure, if you feel it would make the test better. We could write:

`println!("Reference value ID received: {}", reference_id);`

`reference_id` would come from parsing the registration JSON.

Reply:
Yep. BTW this is not a test, this is an application I used to register the launch measurement.
We can also remove it if `keybroker` will provide its own tool.